tests/trainer/resources/sft.ipynb

```
@@ -0,0 +1,640 @@
+    "import shutil\n",
+    "import socket\n",
+    "import time\n",
+    "import json\n",
+    "import random\n",
+    "\n",
+    "import boto3\n",
+    "from botocore.config import Config as BotoConfig\n",
+    "from botocore.exceptions import ClientError\n",
+    "\n",
+    "# --- Global networking safety net: cap all socket operations ---\n",
+    "socket.setdefaulttimeout(10)  # seconds\n",
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)